src/components/image-plus.js

```diff
@@ -135,41 +215,155 @@ errorImage.onload = () => {
   errorTexture.needsUpdate = true;
 };
 
-AFRAME.registerComponent("image-plus", {
+AFRAME.registerComponent("media-video", {
   schema: {
     src: { type: "string" },
-    contentType: { type: "string" },
+    time: { type: "number" },
+    videoPaused: { type: "boolean" },
+    tickRate: { default: 1000 }, // ms interval to send time interval updates
+    syncTolerance: { default: 2 }
```

Comment:
We may find we need to loosen the `syncTolerance`, especially given a `tickRate` of 1000 ms. If we run into trouble finding a tolerance that works well for variable network conditions, we can try increasing the `syncTolerance` anytime we are outside the range and require a seek. Then we can gradually shrink the tolerance as time goes on without being outside the bounds.

Reply:
Yeah I think upping both the tickrate and syncTolerance would be fine.